packages/@react-native-windows/cli/src/runWindows/utils/deploy.ts

```diff
@@ -381,9 +381,18 @@ export async function deployToDesktop(
   } else {
     // Deploy from layout
     // If we have DeployAppRecipe.exe, use it (start in 16.8.4, earlier 16.8 versions have bugs)
+
+    // const configWindows = config.platforms.windows as ConfigWindows;
+    // const deployProjectName = useWinUI3
+    //   ? `${projectName} (Package)`
+    //   : projectName;
+
+    console.log(JSON.stringify(config));
+
+    const deployProjectName = projectName;
     const appxRecipe = path.join(
       path.dirname(appxManifestPath),
-      `${projectName}.build.appxrecipe`,
+      `${deployProjectName}.build.appxrecipe`,
     );
```

Comment:
Revert?

Reply:
@asklar I think this was from one of your commits, are we ok to revert?

Reply:
yes